core/templates/services/stateful/focus-manager.service.spec.ts

```diff
@@ -83,12 +83,32 @@ describe('Focus Manager Service', () => {
     }
   }));
 
-  it('should set focus without scrolling', fakeAsync(() => {
-    spyOn(focusManagerService, 'setFocus');
-    spyOn(windowRef.nativeWindow, 'scrollTo');
-    focusManagerService.setFocusWithoutScroll(focusLabel);
-    flush();
-    expect(focusManagerService.setFocus).toHaveBeenCalledWith(focusLabel);
-    expect(windowRef.nativeWindow.scrollTo).toHaveBeenCalledWith(0, 0);
-  }));
+  it('should set focus without scrolling when schema based list editor is not' +
+  'active', fakeAsync(
+    () => {
+      spyOn(focusManagerService, 'setFocus');
+      spyOn(windowRef.nativeWindow, 'scrollTo');
+      focusManagerService.schemaBasedListEditorIsActive = false;
+
+      focusManagerService.setFocusWithoutScroll(focusLabel);
+      flush();
+      expect(focusManagerService.setFocus).toHaveBeenCalledWith(focusLabel);
+      expect(windowRef.nativeWindow.scrollTo).toHaveBeenCalledWith(0, 0);
+    })
+  );
+
+  it('should set focus without scrolling to top when scheam based lsit editor' +
```

Comment:
One nit otherwise LGTM!
```suggestion
  it('should set focus without scrolling to top when schema based list editor' +
```